### PR TITLE
Bump all dev deps, relax dependabot and remove yarn workspace

### PR DESCRIPTION
**Pull request by Fenntasy**
## Description

This PR changes how we deal with dependencies in this repo.
First it relaxes the dependabot notifications by only notifying us for production deps.
Second, let's be honest and assume we don't really know how monorepos work and remove the yarn workspaces (we'll maybe reintroduce them when we understand how to make them work but right now, they make weird dependabot PRs that does not touch the global `yarn.lock`).
Third, it bumps all dev dependencies and updates some files to reflect his (either a bump in TS or a change in Jest)

## How Has This Been Tested?

Ran each tests to be sure

## Types of changes

- [x] Chore (non-breaking change which refactors / improves the existing code base)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
  change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] My change requires a change to a package version.
- [ ] I have updated the `package.json` version accordingly.
- [ ] I have updated the `CHANGELOG.md` version accordingly.
- [x] I have read the [**CONTRIBUTING**][CONTRIBUTING_FILE] document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

[CONTRIBUTING_FILE]: https://github.com/fewlinesco/node-web-libraries/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: migration/cli.ts [hunk 1]
```diff
@@ -173,7 +173,7 @@ const createCommand = {
       ? { migration: { dirPath: argv.migrationPath } }
       : {};
     const config = await _loadConfig(argv.configPath, overrides);
-    return createMigrationFile(
+    await createMigrationFile(
```
Comment (Fenntasy) [inline migration/cli.ts:176]: the command type accepts only `void | Promise<void>`

File: packages/logging/src/in-memory-logger.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Logger, Metadata } from "@fwl/logging";
+import { Logger, Metadata } from "./logger";
```
Comment (Fenntasy) [inline packages/logging/src/in-memory-logger.ts:1]: This worked before because of yarn workspaces and I have no idea how it worked

Comment (TheRealAstoo) [reply packages/logging/src/in-memory-logger.ts:1]: I think that I've seen something about that. I'll try to find the link

File: packages/web/tests/middlewares/logging.test.ts [hunk 3]
```diff
@@ -38,7 +38,6 @@ test("logs a good result", async () => {
   expect(logger.getLog(0)).toMatchObject({
     message: "",
     path: "/test",
-    remoteaddr: "127.0.0.1",
```
[the remark this answers is not in the document]
Comment (TheRealAstoo) [reply packages/web/tests/middlewares/logging.test.ts:41]: Thank you for the answer

**Review by TheRealAstoo — APPROVED** [review]

**Pull request merged**